ndk/src/relay/subscriptions.ts

```diff
@@ -366,35 +379,38 @@ export class NDKRelaySubscriptions {
         const subId = generateSubId(subscriptions, mergedFilters);
         groupedSubscriptions.req = mergedFilters;
 
-        const subOptions: SubscriptionOptions = { id: subId };
-        if (this.ndkRelay.trusted || subscriptions.every((sub) => sub.opts.skipVerification)) {
-            subOptions.skipVerification = true;
-        }
+        const subOptions: SubscriptionOptions = { 
```

Comment:
This is the other file that required some significant changes because of nostr-tools v2. Take a close look.